types/jest/index.d.ts

```diff
@@ -209,6 +209,14 @@ declare namespace jest {
         readonly name: string;
     }
 
+    interface Each {
+        (cases: any[]): (name: string, fn: (...args: any[]) => any) => void;
```

Comment:
I wonder if this `Each` could be made templated? Is there a way to enforce that If we define the cases as `number[]`, a case with `["foo"]` would be blocked?

Not a blocker for this PR at all, just curious.

Reply:
I was thinking about it too. I only have problem with arrays containing elements of different types. For that Typescript have `Tuples`. So it could be used like that:
```
it.each<[number, string]>([[1, 'one']])
```